testlib/src/test/java/com/diffplug/spotless/generic/LicenseHeaderStepTest.java

```diff
@@ -38,25 +38,56 @@
 	private static final String KEY_FILE_WITHOUT_LICENSE = "license/FileWithoutLicenseHeader.test";
 	private static final String KEY_FILE_WITH_LICENSE_AND_PLACEHOLDER = "license/FileWithLicenseHeaderAndPlaceholder.test";
 
-	// If this constant changes, don't forget to change the similarly-named one in
-	// plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java as well
+	// If these constants change, don't forget to change the similarly-named ones in
+	// plugin-gradle/src/main/java/com/diffplug/gradle/spotless/JavaExtension.java and TODO as well
 	private static final String LICENSE_HEADER_DELIMITER = "package ";
+	private static final String LICENSE_YEAR_DELIMITER = "-";
 
```

Comment:
Rather than introducing changes to the tests, it would be better to use the backward-compatible LicenseHeaderStep.  Not a big deal, fine to leave these as they are, just a tip for the future.  When adding a feature, if you can add a default parameter and thus avoid changing a bunch of tests, just add a default parameter.  The less things change, the easier it is for code-reviewers to verify that there aren't regressions.

Reply:
done